### ISSUE TITLE
Arrow heads missing in element compounds

### ISSUE THREAD

**Issue by sephalon** [cdelker/schemdraw#49]
When adding an element with arrows to an element compound, the arrow heads are missing in the drawing, e.g.:

```python
class SwitchElementCompound(elm.ElementCompound):
    def __init__(self, **kwargs):
        super().__init__(**kwargs)

    def setup(self):
        self.add(elm.SwitchSpdt(action="open"))


with schemdraw.Drawing(file="test.svg") as drawing:
    SwitchElementCompound()
```

![test](https://github.com/user-attachments/assets/d2f0c8f9-1a3f-4d14-912f-d1268da221fb)

Tested with SVG backend only.

**Comment by cdelker** [comment]
Should be fixed in 755a4435383f54333bee6f3c64e9059aaeecf8dd. It was an issue with arrowheads placed on arcs.

**Comment by sephalon** [comment]
@cdelker Thanks!
